crates/ruff_formatter/src/format_element/tag.rs

```diff
@@ -64,7 +64,7 @@ pub enum Tag {
     EndEntry,
 
     /// Delay the printing of its content until the next line break
-    StartLineSuffix,
+    StartLineSuffix(u32),
```

Comment:
Let's use a named field and add some documentation 
```suggestion
    StartLineSuffix { reserved_width: u32 },
```

Reply:
Oh nice. Definitely better.